roles/lib_openshift/library/oc_adm_registry.py

```diff
@@ -2110,6 +2118,53 @@ def add_portal_ip(self, pip):
         '''add cluster ip'''
         self.put(Service.portal_ip, pip)
 
+    def get_external_ips(self):
+        ''' get a list of external_ips '''
+        return self.get(Service.external_ips) or []
```

Comment:
Is there a reason to not use this instances attribute: ``self.external_ips``? Or are you following the pre set pattern in the file?